activerecord/lib/active_record/test_fixtures.rb

```diff
@@ -43,9 +43,11 @@ def set_fixture_class(class_names = {})
       def fixtures(*fixture_set_names)
         if fixture_set_names.first == :all
           raise StandardError, "No fixture path found. Please set `#{self}.fixture_path`." if fixture_path.blank?
-          fixture_set_names = Dir[::File.join(fixture_path, "{**,*}/*.{yml}")].uniq
-          fixture_set_names.reject! { |f| f.start_with?(file_fixture_path.to_s) } if defined?(file_fixture_path) && file_fixture_path
-          fixture_set_names.map! { |f| f[fixture_path.to_s.size..-5].delete_prefix("/") }
+          fixture_set_names = Array(fixture_path).flat_map do |path|
+            names = Dir[::File.join(path, "{**,*}/*.{yml}")].uniq
+            names.reject! { |f| f.start_with?(file_fixture_path.to_s) } if defined?(file_fixture_path) && file_fixture_path
+            names.map! { |f| f[path.to_s.size..-5].delete_prefix("/") }
+          end.uniq
```

Comment:
I am not sure if globbing in a loop like this is unacceptable. I could join the fixture paths here and do one glob, like I do in `fixtures.rb` above. However, we need to know the specific path name of the file on line 49, to remove it from the full file name. I paired with @gmcgibbon on an alternative approach but we didn't find anything to our liking.